.bazelrc.common

```diff
@@ -0,0 +1,26 @@
+# Bazel settings that apply to this repository.
+# Take care to document any settings that you expect users to apply.
+# Settings that apply only to CI are in .github/workflows/ci.bazelrc
+
+# Allow the Bazel server to check directory sources for changes.
+# Recommended when using copy_directory, see
+# https://github.com/aspect-build/bazel-lib/blob/main/docs/copy_directory.md
+startup --host_jvm_args=-DBAZEL_TRACK_SOURCE_DIRECTORIES=1
+
+# In general, the rules in this repository assume that runfiles
+# are enabled as we do not support no runfiles case.
+#
+# If you are developing on Windows, you must either run bazel
+# with administrator priviledges or enable developer mode. If
+# you do not you may hit this error on Windows:
+#
+#   Bazel needs to create symlinks to build the runfiles tree.
+#   Creating symlinks on Windows requires one of the following:
+#       1. Bazel is run with administrator privileges.
+#       2. The system version is Windows 10 Creators Update (1703) or later
+#          and developer mode is enabled.
+build --enable_runfiles
+
+# Turn off legacy external runfiles
+# This prevents accidentally depending on this feature, which Bazel will remove.
+build --nolegacy_external_runfiles
```

Comment:
not all users can turn this on since it doesn't work with some rules. I'd recommend not telling users to do this - we should just enforce it on ourselves